scripts/cibuild.sh

```diff
@@ -54,10 +54,18 @@ if [[ ! -z $SYNTAX ]]; then
 fi
 
 echo -e "\n>> Running unit tests..."
-invoke unittest --failfast
+invoke unittest --failfast --buffer --keepdb
 RC=$?
 if [[ $RC != 0 ]]; then
-	echo -e "\n$(info) one or more tests failed, failing build."
+	echo -e "\n$(info) one or more unit tests failed, failing build."
+	exit $RC
+fi
+
+echo -e "\n>> Running integration tests..."
+invoke unittest --failfast --buffer --keepdb --label nautobot.core.tests.integration
```

Comment:
Shouldn't `NAUTOBOT_INTEGRATION_TEST` be explicitly set in this case?

Reply:
Yes, but we need to come up with a way to make sure that environment variables are passed thru from shell -> invoke -> Docker